base/compiler/ssair/show.jl

```diff
@@ -104,45 +116,63 @@ function show_unquoted_phinode(io::IO, stmt::PhiNode, indent::Int, prefix::Strin
     print(io, ')')
 end
 
-function show_unquoted(io::IO, stmt::PhiCNode, indent::Int, ::Int)
+function show_unquoted(io::IO, stmt::PhiCNode, indent::Int, prec::Int, unstable_ssa::BitSet = BitSet())
     print(io, "φᶜ (")
     first = true
     for v in stmt.values
         first ? (first = false) : print(io, ", ")
-        show_unquoted(io, v, indent)
+        if v isa SSAValue
+            show_unquoted(io, v, indent, prec, (v in unstable_ssa) ? true : false)
+        else
+            show_unquoted(io, v, indent, prec)
+        end
     end
     print(io, ")")
 end
 
-function show_unquoted(io::IO, stmt::PiNode, indent::Int, ::Int)
+function show_unquoted(io::IO, stmt::PiNode, indent::Int, prec::Int, unstable_ssa::BitSet=BitSet())
     print(io, "π (")
-    show_unquoted(io, stmt.val, indent)
+    show_unquoted(io, stmt.val, indent, prec, (stmt.val.id in unstable_ssa) ? true : false)
```

Comment:
So previously `prec` was ignored, now it is passed on. Is this a necessary for this patch? (I am not objecting, just wondering whether this fixes something, resp. where it might break something else)